docs/evm/ethereum-vs-flow-evm.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Differences between EVM on Flow and Ethereum
```

Comment:
Given that we already have docs on the difference between EVM that still need to be refactored (https://docs-git-ramtin-821-onflow.vercel.app/build/differences-vs-evm) and the content I would recommend renaming this doc to 'How it Works' or 'How EVM on Flow Works'.

Reply:
updated